modules/Settings/WebserviceUsers/models/Record.php

```diff
@@ -340,4 +353,39 @@ public function delete()
 		}
 		return !empty($result);
 	}
+
+	/**
+	 * Send mails with access.
+	 *
+	 * @return void
+	 */
+	public function sendEmail()
+	{
+		if (empty($this->get('crmid'))) {
+			return;
+		}
+		$moduleName = 'Contacts';
+		$recordModel = Vtiger_Record_Model::getInstanceById($this->get('crmid'), $moduleName);
+		$moduleModel = Vtiger_Module_Model::getInstance($moduleName);
+		$emailsFields = $moduleModel->getFieldsByType('email');
```

Comment:
check permission to send emails (field in contact)
```suggestion
		$emailsFields = $recordModel->getModule()->getFieldsByType('email');
```